### PR TITLE
Feature: UTCT-21 Roles and Auth

### PR DESCRIPTION
**Pull request by UTCGilligan**

### CODE REVIEW

**Review by majorschwartz — COMMENTED** [review]
Looks good.

File: .lando.yml [hunk 1]
```diff
@@ -5,3 +5,8 @@ config:
   webroot: .
   database: mariadb
   xdebug: false
+services:
```
Comment (UTCGilligan) [inline .lando.yml:8]: Added because the SAML plugin requires a web server alias.

File: composer.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
         "source": "https://github.com/YOURLS/YOURLS"
     },
     "require": {
-        "php": ">=7.4",
+        "php": ">=8.1",
```
Comment (UTCGilligan) [inline composer.json:19]: App probably will run on PHP 8.2, but needs testing in a hosted environment.

File: config/default-ssl.conf [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# Default Virtual Host configuration.
```
Comment (UTCGilligan) [inline config/default-ssl.conf:1]: Lando local httpd config.

File: config/httpd.conf [hunk 4]
```diff
@@ -0,0 +1,520 @@
+#
```
Comment (UTCGilligan) [inline config/httpd.conf:1]: Lando local httpd config.

File: user/plugins/yourls-saml/.github/dependabot.yml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
```
Comment (UTCGilligan) [inline user/plugins/yourls-saml/.github/dependabot.yml:1]: Good 'ol Dependabot... how we hate him.

File: user/plugins/yourls-saml/README.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
+# YOURLS SAML Authentication
```
Comment (UTCGilligan) [inline user/plugins/yourls-saml/README.md:1]: Hopefully you have actually read this?

**Review by UTCGilligan — COMMENTED** [review]
All files reviewed. Looks good. 

Working on the SAML auth config with Carson Coe.

**Pull request merged**